manifests/m/Microsoft/ReportBuilder/15.0.19914.0/Microsoft.ReportBuilder.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.ReportBuilder
+PackageVersion: 15.0.19914.0
+PackageLocale: en-US
+Publisher: Microsoft
+PackageName: ReportBuilder

```

```suggestion
PackageName: Microsoft Report Builder
```
This must match Control Panel.